configs/eln_extras_meta.yaml

```diff
@@ -39,7 +39,6 @@ data:
     - libxcrypt-compat
     - linux-sysinfo-snapshot
```

Comment:
Did you mean to remove this from here while adding it to the new config?

Reply:
uh... thanks, yes I did

Reply:
updated, thanks